modules/gentemplates/swagger-codegen/src/test/java/com/reprezen/genflow/swagger/codegen/SwaggerCodegenModulesTest.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright © 2013, 2016 Modelsolv, Inc.
```

Comment:
This is a new file. Should we change this to 2019?

Reply:
@nbhusare , Yes, we should update our copyright notices to say 2013-2019:

>  Copyright © 2013, 2016 Modelsolv, Inc.

I'll open a Jira for this so we can make this change across all of our open source projects.